cmd/dra-example-kubeletplugin/state.go

```diff
@@ -222,6 +228,7 @@ func (s *DeviceState) syncPreparedDevicesToCRDSpec(spec *nascrd.NodeAllocationSt
 				}
 				prepared.Gpu.Devices = append(prepared.Gpu.Devices, outdevice)
 			}
+		default:
```

Comment:
Error?

Reply:
Just loggging, when implemented, there's nothing that expects an error to be returned from here or parent, unsupported devices are ignored and logged.

Reply:
As above, let's change the function so that it can return an error.

Reply:
Call it defensive programming. Mistakes happen.